webapp/src/pages/reviewAssignment/components/ReviewAssignmentComponent.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
+import { reviewService } from "../../../services/reviews";
+import { withRouter } from "react-router";
+import ReactTable from 'react-table'
+
+import 'react-table/react-table.css'
+
+import FormTextBox from "../../../components/form/FormTextBox";
+
+const DEFAULT_EVENT_ID = process.env.DEFAULT_EVENT_ID || 1;
+
+class ReviewAssignmentComponent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: true,
+      reviewers: null,
+      error: "",
+      newReviewerEmail: "",
+      reviewSummary: {}
+    };
+    this.renderEditable = this.renderEditable.bind(this);
```

Comment:
This will work, but removing this line and making the function an arrow function will be cleaner.
```renderEditable(cellInfo)``` to ```renderEditable = cellInfo =>```